src/main/java/training/Elbow/SimulationElbow.java

```diff
@@ -0,0 +1,34 @@
+package training.Elbow;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+import utils.GBSubsystem;
+import utils.simulation.SingleJointedArmSimulation;
+
+import static training.Elbow.ElbowConstants.*;
+
+public class SimulationElbow implements IElbow {
+    private final SingleJointedArmSimulation arm;
+
+    public SimulationElbow() {
+        this.arm = new SingleJointedArmSimulation(new SingleJointedArmSim(ELBOW_GEARBOX, ELBOW_GEARING, SingleJointedArmSim.estimateMOI(ELBOW_LENGTH_METERS, ELBOW_MASS), ELBOW_LENGTH_METERS, ELBOW_MINIMUM_ANGLE.getRadians(), ELBOW_MAXIMUM_ANGLE.getRadians(), true, 0));
```

Comment:
add apply CoNFIG, learn about it

Reply:
can you give me a specific name since I can't find anything related to DCmotor and config in wpilib